### PR TITLE
Update React to 15.5.3

### PR DESCRIPTION
**Pull request by airamrguez**

### CODE REVIEW

**Comment by misterjunio** [comment]
@airamrguez I tried to merge but `yarn install` timed out when fetching the `react-dom-15.5.3` package, which did not happen locally for me with `npm`. Do you have any idea what the issue might be? Did you use `yarn`?

**Comment by airamrguez** [comment]
I used yarn and I did not have any problems installing the project dependencies. May be `rm -rf node-modules && yarn` fixes it?

**Comment by misterjunio** [comment]
@airamrguez hum I tried locally and it timed out in a different package, `prop-types-15.5.6`. Which version of `yarn` are you using?

**Comment by frandiox** [comment]
@airamrguez @misterjunio  I think I found the issue. Not sure why but the `yarn.lock` file in this PR had some URLs changed to an IP, and this IP was timing out. I will revert the revert :D

```
 -react@^15.4.2:
 -  version "15.4.2"
 -  resolved "https://registry.yarnpkg.com/react/-/react-15.4.2.tgz#41f7991b26185392ba9bae96c8889e7e018397ef"
 +react@^15.5.3:
 +  version "15.5.3"
 +  resolved "http://192.168.1.199:4873/react/-/react-15.5.3.tgz#84055382c025dec4e3b902bb61a8697cc79c1258"
```

**Comment by airamrguez** [comment]
@frandiox good catch! I fogot. I use npm behind a proxy. Nice to see this pr merged.

**Comment by frandiox** [comment]
@airamrguez And by the way, thanks for also updating `react-hot-loader` :)
I'll try to have a look at the other PRs soon.

**Comment by airamrguez** [comment]
ok, great! thanks @frandiox

File: bindings/react/rollup.config.js [hunk 1]
```diff
@@ -26,13 +26,15 @@ export default {
     'react',
     'react-dom/server',
     'react-dom',
-    'onsenui'
+    'onsenui',
+    'prop-types'
```
Comment (asial-matagawa) [inline bindings/react/rollup.config.js]: Sorry, I mean this line.

**Pull request merged**